shoes-core/lib/shoes/builtin_methods.rb

```diff
@@ -52,7 +52,7 @@ def ask_open_folder
     end
 
     def ask_save_folder
-      Shoes::Dialog.new.dialog_chooser 'Save Folder...', :folder
+      Shoes::Dialog.new.dialog_chooser 'Save Folder...', :folder, :save
```

Comment:
I almost commented saying that instad of `:save` we could pass in the style directly. But this is in `shoes-core` and therefore rightfully SWT free - good catch and engineering! :)